app/build.gradle

```diff
@@ -70,6 +71,14 @@ dependencies {
         exclude module: 'recyclerview-v7'
         exclude module: 'appcompat-v7'
     }
+
+    // Import the BoM for the Firebase platform
+    implementation platform('com.google.firebase:firebase-bom:26.7.0')
+    // Declare the dependency for the Firebase Authentication library
+    implementation 'com.google.firebase:firebase-auth-ktx'
+    // Declare the dependency for the Realtime Database library
+    implementation 'com.google.firebase:firebase-database-ktx'
+    androidTestImplementation 'androidx.test:core-ktx:1.1.0'
```

Comment:
Cirrus says that the version 1.3.0 is available, might be nice to update !

Reply:
Right I'll change that in the next branch I push :)